### PR TITLE
fix: api parameters

### PR DESCRIPTION
**Pull request by guanbinrui**
## Description

Closes [MF-209](https://mask.atlassian.net/browse/MF-209)

## Type of change

- [ ] Documentation
- [ ] Code refactoring (Restructuring existing code w/o changing its observable behavior)
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (a fix or feature that would make something no longer possible to do/require old user must upgrade their Mask Network to this new version)

### CODE REVIEW

**Comment by guanbinrui** [comment]
@DimensionDev/qa

File: packages/dashboard/src/index.tsx [hunk 1]
```diff
@@ -16,7 +16,7 @@ if (import.meta.webpackHot) {
 }
 
 createRoot(document.getElementById('root')!).render(
-    <StrictMode>
```
Comment (Jack-Works) [inline packages/dashboard/src/index.tsx]: Reason?

File: packages/shared-base/src/Pure/index.ts [hunk 2]
```diff
@@ -1,3 +1,5 @@
 export const EMPTY_LIST = Object.freeze([]) as never[]
 
 export const EMPTY_OBJECT = Object.freeze({})
+
+export const NOOP = Object.freeze(() => {})
```
Comment (Jack-Works) [inline packages/shared-base/src/Pure/index.ts]: use `noop` from lodash.

File: cspell.json [hunk 3]
```diff
@@ -67,7 +67,6 @@
         "datas",
         "debank",
         "denorm",
-        "destory",
```
Comment (Jack-Works) [inline cspell.json]: I suggest you to open another PR for this correction.

**Review by Jack-Works — CHANGES_REQUESTED** [review]

**Review by Jack-Works — DISMISSED** [review]

**Review by Jack-Works — APPROVED** [review]

**Pull request closed without merging**